lib/v2/nua/index.js

```diff
@@ -0,0 +1,23 @@
+const util = require('./utils');
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type;
+
+    const baseUrl = 'https://www.nua.edu.cn';
+    const newsUrl = `${baseUrl}/${type}/list.htm`;
+    const listName = 'li.news';
+    const listDate = '.news_meta';
+    const webPageName = '.col_title';
+
+    const artiContent = '.read';
+    const items = await util.ProcessList(newsUrl, baseUrl, listName, listDate);
+    const pageName = await util.ProcessPageName(newsUrl, webPageName);
```

Comment:
Please combine these two functions. You are essentially visiting the same webpage twice.
https://github.com/DIYgod/RSSHub/runs/8062068148?check_suite_focus=true#step:11:15